src/dev/i18n/tasks/check_configs.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { resolve, join } from 'path';
+import { I18N_RC } from '../constants';
+import { ErrorReporter, checkConfigNamespacePrefix, arrayify } from '..';
+
+export function checkConfigs(additionalConfigPaths: string | string[] = []) {
+  const root = join(__dirname, '../../../../');
+  const kibanaRC = resolve(root, I18N_RC);
+  const xpackRC = resolve(root, `x-pack/${I18N_RC}`);
```

Comment:
resolve takes multiple arguments so that we don't have to concatenate with possibly the wrong path separator, even though it would take care of replacing it. I think it's best to break the habit of using string concatenation when dealing with paths as much as possible and use `resolve(root, 'x-pack', I18N_RC)` instead.